src/state/symbols/utils/index.js

```diff
@@ -8,7 +8,7 @@ export * from './mapping'
 // BTCUSD -> tBTCUSD
 // BTCF0:USDF0 -> tBTCF0:USDF0
 // USD -> fUSD
-const addPrefix = (symbol = '', isFunding = false) => (
+export const addPrefix = (symbol = '', isFunding = false) => (
```

Comment:
So i would definitely change this. 
What would do is the following: 
1 - Add t to each result from pairs, that should be the trading pair. 
```console
const tradingPairs = pairs.map(p => `t${p}`)
```